packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -459,7 +459,7 @@ class Slider extends React.Component {
       [classes.rtl]: theme.direction === 'rtl',
     };
 
-    const className = classNames(
+    const className = clsx(
```

Comment:
I would've preferred to simply rename the import. In general doesn't really matter from where the implementation for the interface comes from. This on specific just makes it harder to parse by a human. I just don't see the arguments of mangling names.